pact/provider/pact-broker-loader/src/main/java/org/arquillian/algeron/pact/provider/core/loader/pactbroker/PactBrokerLoader.java

```diff
@@ -59,36 +58,32 @@
             .build();
     private Callable<HttpResponse> httpResponseCallable;
 
-    public PactBrokerLoader(final String pactBrokerHost, final String pactBrokerPort, final String pactBrokerProtocol) {
-        this(pactBrokerHost, pactBrokerPort, pactBrokerProtocol, Collections.singletonList("latest"));
+    public PactBrokerLoader() {
+        super();
     }
 
-    public PactBrokerLoader(final String pactBrokerHost, final String pactBrokerPort, final String pactBrokerProtocol,
-                            final List<String> tags) {
-        this.pactBrokerHost = pactBrokerHost;
-        this.pactBrokerPort = pactBrokerPort;
-        this.pactBrokerProtocol = pactBrokerProtocol;
-        this.pactBrokerTags = tags;
+
+    @Override
+    public void setProviderName(String providerName) {
+        this.providerName = providerName;
     }
 
-    public PactBrokerLoader(final PactBroker pactBroker) {
-        this(getResolvedValue(pactBroker.host()),
-                getResolvedValue(pactBroker.port()),
-                getResolvedValue(pactBroker.protocol()),
-                        Arrays.stream(pactBroker.tags())
-                                .map(tag -> getResolvedValue(tag))
-                                .collect(toList()));
+    @Override
+    public void configure(Map<String, Object> configuration) {
+        this.pactBroker = new PactBrokerImpl(configuration);
```

Comment:
But it holds some default behaviour driven from configuration - can't we think about the name revealing this functionality instead?